ga.rpmtw.rpmlauncher.yml

```diff
@@ -0,0 +1,39 @@
+app-id: ga.rpmtw.rpmlauncher
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: RPMLauncher
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # Needs to talk to the network:
+  - --share=network
+  # Needs to save config files
+  - --filesystem=home
```

Comment:
This shouldn't be needed. By default, flatpak applications have write access to $XDG_DATA_HOME and $XDG_CONFIG_HOME – as long as the app respect these environment variables, there's no need for sandbox hole.

Reply:
Yes, but this application allows users to customize the data storage location and therefore requires filesystem permissions.

Reply:
I would at least change this line to `--persist=RPMLauncher` in that case. People can grant wider filesystem permissions via CLI or Flatseal.

Reply:
ok